tracking-android/src/main/java/com/visenze/usertrackingsdk/UserEventTracker.java

```diff
@@ -2,11 +2,13 @@
 
 import android.content.Context;
 
+import java.util.HashMap;
+
 /**
  * Created by visenze on 26/2/16.
  */
 public class UserEventTracker {
-    private String accessKey;
+    private HashMap<String, String> map;
```

Comment:
Try to avoid naming a parameter as 'map', might cause confusion. Let's name it as 'initParams'